src/vdom/diff.js

```diff
@@ -293,6 +293,7 @@ export function removeChildren(node) {
  */
 function diffAttributes(dom, attrs, old) {
 	let name;
+	const isInput = dom.tagName.toLowerCase() === "input";
```

Comment:
Need to use `dom.normalizedNodeName || dom.nodeName` here

Reply:
Ah, thanks for taking a look. Realized `isNamedNode` is there for this exact purpose, so I dropped that in place of the line in question.